### ISSUE TITLE
Does this work on CentOS 7?

### ISSUE THREAD

**Issue by Yzord** [ossec/ossec-hids#733]
Would love to know if this works with CentOS 7.2. Anyone an idea?

**Comment by ddpbsd** [comment]
Have you had problems with it?

**Comment by aremai** [comment]
do you mean anything specific or OSSEC in general?

**Comment by Yzord** [comment]
I didn't tried it, because it would come on a production server. And CentOS7 isn't on the list of supported distros

**Comment by aquerubin** [comment]
On Tue, 19 Jan 2016, Yzord wrote:

> I didn't tried it, because it would come on a production server. And 
> CentOS7 isn't on the list of supported distros

The current master branch builds/runs fine on RHEL7.1 based distros 
(Scientific Linux in particular).  I would expect 7.2 wouldn't be any 
different.  You do need to add the firewalld-based responder to ossec.conf 
instead of the iptables-based responders.  And if you're using logrotate 
you do need to modify the file labels in /var/ossec/logs accordingly since 
SELinux comes enabled by default.  Otherwise it runs the same as on RHEL6 
based distros.

Antonio Querubin
e-mail:  tony@lavanauts.org
xmpp:  antonioquerubin@gmail.com

**Comment by aremai** [comment]
I think CentOS/RHEL 7 first introduced systemd/journald.
It's easy to manage the ossec-control start/stop script and the individual ossec processes through systemd.
But, is analysisd also able to analyse journald output? On my box I have journald configured to send its log output through rsyslog (which kinda defeats the purpose of journald).
Is there a better way to have OSSEC analyse journald logs?

**Comment by ddpbsd** [comment]
Ossec doesn't support journald, only the proper syslog output. I'm hoping
the linux distros see the light before we have to add support for it.

**Comment by aremai** [comment]
well, this says something else ;)
https://github.com/ossec/ossec-hids/blob/master/etc/rules/systemd_rules.xml

**Comment by ddpbsd** [comment]
Difference of opinion. Supporting the logs (via syslog) is different than
supporting journald (polling journald for the logs).

**Comment by aremai** [comment]
that is true. that was a misunderstanding from my side, sorry about that.
however, I have decided to look into writing a ruleset for journald.
i think it is very unlikely that the major linux distributions will refrain from using journald in the future.
this problem needs to be tackled somehow, because the current releases (centos, debian, ubuntu) all use journalctl/journald for its system logging.
so in other words: writing a decoder / ruleset is on my top priority list...

**Comment by ddpbsd** [comment]
Writing rules is easy, writing the support for logcollector to query
journald is the tough part.

**Comment by aquerubin** [comment]
Update:  I recently upgraded several SL 7.1 servers to SL 7.2.  All are 
running ossec agent with no issues.  CentOS 7.2 should be ok.

Antonio Querubin
e-mail:  tony@lavanauts.org
xmpp:  antonioquerubin@gmail.com

**Comment by axot** [comment]
Does `cis_rhel6_linux_rcl.txt` work for CentOS 7?

**Comment by santiago-bassett** [comment]
Most likely it doesn't, as one of the pre-checks looks for the release version to be 6. 
[CIS - Testing against the CIS Red Hat Enterprise Linux 5 Benchmark v2.1.0] [any required] [http://www.ossec.net/]
f:/etc/redhat-release -> r:^Red Hat Enterprise Linux \S+ release 6;
f:/etc/redhat-release -> r:^CentOS && r:release 6;
f:/etc/redhat-release -> r:^Cloud && r:release 6;
f:/etc/redhat-release -> r:^Oracle && r:release 6;
f:/etc/redhat-release -> r:^Better && r:release 6;

**Comment by ddpbsd** [comment]
@axot You can add a check so that it runs on centos 7 though. If you do, report back with your findings, and we can work on updating it.

**Comment by atomicturtle** [comment]
I just updated these for EL7, and merged in the wazuh PCI markup changes. I should have  a PR ready for this later today or tomorrow.

**Comment by atomicturtle** [comment]
Merged in with https://github.com/ossec/ossec-hids/pull/788
